src/components/auth.js

```diff
@@ -0,0 +1,28 @@
+import { useSearchParams } from 'react-router-dom';
+import axios from 'axios';
+
+
+const Auth = () => {
+  const [searchParams, ] = useSearchParams();
+  const code = searchParams.get('code');
+  const sessionState = searchParams.get('session_state');
+
+  // axios.post('http://localhost:8000/apps/user/v1/auth/', {
+  axios.post('https://talk.gausslabs.ai/api/apps/user/v1/auth/', {
+    code: code,
+    session_state: sessionState,
+  })
+  .then(function (response) {
+    var loginMessage = document.getElementById('login-message');
```

Comment:
I think using let is a better convention for defining variables.
var allows to declare variables with the same name multiple times while let does not allow it.
Using let can help prevent overwriting important variables.
I wasn't aware of this but maybe we should consider let in the future.